proxy/logging/unit-tests/test_LogUtils.cc

```diff
@@ -150,52 +150,4 @@ TEST_CASE("get_unrolled_filename parses possible log files as expected", "[get_u
 
   constexpr ts::TextView no_dot = "logging_yaml";
   REQUIRE(get_unrolled_filename(no_dot) == no_dot);
-}
-
-TEST_CASE("LogUtils pure escapify url", "[pure_esc_url]")
-{
-  char input[][32] = {
-    " ",
-    "%",
-    "% ",
-    "%20",
-  };
-  const char *expected[] = {
-    "%20",
-    "%25",
-    "%25%20",
-    "%2520",
-  };
-  char output[128];
-  int output_len;
-
-  int n = sizeof(input) / sizeof(input[0]);
-  for (int i = 0; i < n; ++i) {
-    LogUtils::pure_escapify_url(NULL, input[i], std::strlen(input[i]), &output_len, output, 128);
-    CHECK(std::string_view(output) == expected[i]);
-  }
-}
-
-TEST_CASE("LogUtils escapify url", "[esc_url]")
-{
-  char input[][32] = {
-    " ",
-    "%",
-    "% ",
-    "%20",
-  };
-  const char *expected[] = {
-    "%20",
-    "%25",
-    "%25%20",
-    "%20",
-  };
-  char output[128];
-  int output_len;
-
-  int n = sizeof(input) / sizeof(input[0]);
-  for (int i = 0; i < n; ++i) {
-    LogUtils::escapify_url(NULL, input[i], std::strlen(input[i]), &output_len, output, 128);
-    CHECK(std::string_view(output) == expected[i]);
-  }
-}
+}
```

Comment:
Github shows that this file doesn't have a trailing newline. Might want to update your editor to add those.

Reply:
Do we require a trailing newline? I wonder if clang-format can correct it.

Reply:
clang-format can do it, but we don't require it.
```
$ git diff .clang-format
diff --git a/.clang-format b/.clang-format
index 7931e13cf..d6431535b 100644
--- a/.clang-format
+++ b/.clang-format
@@ -127,7 +127,7 @@ IndentRequiresClause: true
 IndentWidth:     2
 IndentWrappedFunctionNames: false
 InsertBraces:    false
-InsertNewlineAtEOF: false
+InsertNewlineAtEOF: true
 InsertTrailingCommas: None
 IntegerLiteralSeparator:
   Binary:          0
```
then
```
$ git diff --stat
 .clang-format                                      | 2 +-
 include/shared/rpc/RPCClient.h                     | 2 +-
 include/tscore/Encoding.h                          | 2 +-
 mgmt/rpc/handlers/common/ErrorUtils.h              | 2 +-
 mgmt/rpc/handlers/common/Utils.h                   | 2 +-
 mgmt/rpc/handlers/common/convert.h                 | 2 +-
 mgmt/rpc/handlers/config/Configuration.h           | 2 +-
 mgmt/rpc/server/CommBase.cc                        | 2 +-
 mgmt/rpc/server/CommBase.h                         | 2 +-
 plugins/experimental/slice/prefetch.h              | 2 +-
 plugins/server_push_preload/server_push_preload.cc | 2 +-
 proxy/logging/unit-tests/test_LogUtils.cc          | 2 +-
 src/traffic_ctl/PrintUtils.h                       | 2 +-
 src/traffic_server/RpcAdminPubHandlers.h           | 2 +-
 src/tscore/Encoding.cc                             | 2 +-
 tests/tools/plugins/cert_update.cc                 | 2 +-
 16 files changed, 16 insertions(+), 16 deletions(-)
```

Reply:
Added new line, and made a PR to require new line at eof.
https://github.com/apache/trafficserver/pull/9930